### PR TITLE
Remove fixed seed for LHE generation

### PR DESCRIPTION
**Pull request by selvaggi**
also changed max errors and default number of events to generate

### CODE REVIEW

File: Generation/data/Pythia_LHEinput.cmd [hunk 1]
```diff
@@ -7,10 +7,8 @@
 ! Adjusted from Pythia example: main42.cmnd
 
 ! 1) Settings that will be used in a main program.
-Main:numberOfEvents = 1            ! number of events to generate
-Main:timesAllowErrors = 3          ! abort run after this many flawed events
-Random:seed = 1234                ! initialize random generator with a seed
-
+Main:numberOfEvents = 100          ! number of events to generate
+Main:timesAllowErrors = 100        ! abort run after this many flawed events
```
Comment (clementhelsens) [inline Generation/data/Pythia_LHEinput.cmd]: hum, 100 is maybe a bit too much no?
In a typical job of 10k events we don't expect more than a few failures.

**Comment by jlingema** [comment]
ok to test.

File: Generation/data/Pythia_LHEinput.cmd (hunk 1 above)
Comment (jlingema) [inline Generation/data/Pythia_LHEinput.cmd]: Does this need to be changed or is it ok to merge?

Comment (selvaggi) [inline Generation/data/Pythia_LHEinput.cmd]: no let's keep 100, does not harm. That's only 1% failure in 10k events

Comment (clementhelsens) [inline Generation/data/Pythia_LHEinput.cmd]: ok, indeed it is more safe

**Pull request merged**